test-suite/js/src/test/scala/org/scalajs/testsuite/library/UnionTypeTest.scala

```diff
@@ -12,7 +12,7 @@ import scala.language.implicitConversions
 import scala.collection.mutable
 
 import scala.scalajs.js
-import js.|
+import scala.scalajs.js.{UndefOr, |}
```

Comment:
Do not import `UndefOr`. The recommended coding style (and the one we apply in this codebase) is to use `js.UndefOr` at use site.